tensorflow/python/keras/metrics.py

```diff
@@ -770,6 +770,8 @@ def __init__(self, name='categorical_accuracy', dtype=None):
 class SparseCategoricalAccuracy(MeanMetricWrapper):
   """Calculates how often predictions matches integer labels.
 
+  `acc = np.dot(sample_weight, np.equal(y_true, np.argmax(y_pred, axis=1))`
+
```

Comment:
I would recommend the following

```suggestion
  ```python
  acc = np.dot(sample_weight, np.equal(y_true, np.argmax(y_pred, axis=1))
  ```
```

Reply:
Thank you for the feedback. I have incorporated the changes into the PR.